src/CoreWCF.Primitives/src/CoreWCF/Channels/MessageHeaders.cs

```diff
@@ -237,7 +237,7 @@ public EndpointAddress From
             }
         }
 
-        internal bool HasMustUnderstandBeenModified
+        public bool HasMustUnderstandBeenModified
         {
```

Comment:
I would prefer to go a different route than this. There is a UnderstoodHeaders property on this class which is of type UnderstoodHeaders which are both public on NetFx, but not in CoreWCF. Make this property and the type public. Then on the UnderstoodHeaders class, there's a Modified property which is internal. It makes sense to make that public even though it's internal on NetFx. The callsite of this property in WebHttp will then use MessageHeaders.UnderstoodHeaders.Modified instead.

Reply:
This has been addressed.